platform/base.go

```diff
@@ -47,13 +47,14 @@ type BaseCluster struct {
 	rconf      *RuntimeConfig
 	platform   Name
 	ctPlatform string
+	baseopts   *Options
 }
 
-func NewBaseCluster(basename string, rconf *RuntimeConfig, platform Name, ctPlatform string) (*BaseCluster, error) {
-	return NewBaseClusterWithDialer(basename, rconf, platform, ctPlatform, network.NewRetryDialer())
+func NewBaseCluster(opts *Options, rconf *RuntimeConfig, platform Name, ctPlatform string) (*BaseCluster, error) {
```

Comment:
I'm afraid this time I'm going to insist: this refactor should be in a separate commit.

Reply:
Fixed now